src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -361,14 +362,58 @@ invalidated when a device disconnects. This means your code should always
+characteristic value. You can access them, read and write to in a similar way
+to Bluetooth GATT characteristics.
+
+Let's see for instance how to read the user description of the measurement
+interval of the device's health thermometer.
+
+In the example below, `health_thermometer` is the [Health
+Thermometer Service](https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.service.health_thermometer.xml), `measurement_interval` the [Measurement Interval Characteristic](https://developer.bluetooth.org/gatt/characteristics/Pages/CharacteristicViewer.aspx?u=org.bluetooth.characteristic.measurement_interval.xml), and `gatt.characteristic_user_description` the [Characteristic User Description Descriptor](https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.descriptor.gatt.characteristic_user_description.xml).
+
+<pre class="prettyprint">
+navigator.bluetooth.requestDevice({ filters: [{ services: ['health_thermometer'] }] })
+.then(device => device.gatt.connect())
+.then(server => server.getPrimaryService('health_thermometer'))
+.then(service => service.getCharacteristic('measurement_interval'))
+<strong>.then(characteristic => characteristic.getDescriptor('gatt.characteristic_user_description'))
+.then(descriptor => descriptor.readValue())
+.then(value => {
+  let decoder = new TextDecoder('utf-8');
+  console.log('User Description: ' + decoder.decode(value));
+})</strong>
+.catch(error => { console.log(error); });
+</pre>
+
+Now that we've read the user description of the measurement interval of the
+device's health thermometer, let's see how to update it and write a custom
+value.
+
+<pre class="prettyprint">
+navigator.bluetooth.requestDevice({ filters: [{ services: ['health_thermometer'] }] })
+.then(device => device.gatt.connect())
+.then(server => server.getPrimaryService('health_thermometer'))
+.then(service => service.getCharacteristic('measurement_interval'))
+.then(characteristic => characteristic.getDescriptor('gatt.characteristic_user_description'))
+<strong>.then(descriptor => {
+  let encoder = new TextEncoder('utf-8');
+  let userDescription = encoder.encode('Defines the time between measurements.');
+  return descriptor.writeValue(userDescription);
+})</strong>
+.catch(error => { console.log(error); });
+</pre>
+
 ## Samples, Demos and Codelabs
 
 All [Web Bluetooth samples](https://googlechrome.github.io/samples/web-bluetooth/index.html) below
-have been tested with the Web Bluetooth flag enabled. To enjoy these samples to
-their fullest, I recommend you install the
+have been successfully tested. To enjoy these samples to their fullest, I
+recommend you install the
 [BLE Peripheral Simulator Android App](https://play.google.com/store/apps/details?id=io.github.webbluetoothcg.bletestperipheral)
-which simulates a BLE Peripheral with a Battery Service or a Heart Rate
-Service.
+which simulates a BLE Peripheral with a Battery Service, a Heart Rate
```

Comment:
'Peripheral' should be lower case. Are these services actual products. They shouldn't be lower case either.